src/pages/EditRequestPage.js

```diff
@@ -104,18 +123,34 @@ function EditRequestPage({report, route, parentReport, policyCategories, policyT
     // A flag for showing the tags page
     const shouldShowTags = isPolicyExpenseChat && (transactionTag || OptionsListUtils.hasEnabledOptions(lodashValues(policyTagList)));
 
+    const reportAction = ReportActionsUtils.getReportAction(parentReportID, parentReportActionID);
+
+    // For small screen, we don't call openReport API when we go to a sub report page by deeplink
```

Comment:
1) Where are we calling openReport API separately for big screens?
2) If the said is true for small screen devices, shouldn't other types of reports be affected by the same? Like Task and edit Task?